conf/output/test-output-1.json

```diff
@@ -3,7 +3,6 @@
   "type" : [ "BibliographicResource", "Article" ],
   "rpbId" : "929t112201",
   "id" : "https://lobid.org/resources/929t112201",
-  "hbzId" : "CT003010543",
```

Comment:
This seems to be the real hbzId of the resource not the one of the parent publication.